_config.yml

```diff
@@ -25,8 +25,21 @@ gems:
   - jekyll-assets
   - jekyll-sitemap
 
-name: cga_site
+name: cloud.gov.au
 
 permalink: pretty
 
 url: # add site url
+
+global_navigation:
+  - Overview: '/overview/'
+  - Docs: 'http://docs.cloud.gov.au'
```

Comment:
This is a great point. I have added an GitHub issue in this repo to address this https://github.com/AusDTO/cga_site/issues/4